eng/Versions.props

```diff
@@ -32,6 +32,7 @@
     <!-- DotNetCliVersion MUST match the dotnet version in global.json.
          Otherwise, this version of dotnet will not be installed and the build will error out. -->
     <DotNetCliVersion>$([System.Text.RegularExpressions.Regex]::Match($([System.IO.File]::ReadAllText('$(MSBuildThisFileDirectory)..\global.json')), '"dotnet": "([^"]*)"').Groups.get_Item(1))</DotNetCliVersion>
+    <MicrosoftDotNetXUnitExtensionsVersion>6.0.0-beta.21329.2</MicrosoftDotNetXUnitExtensionsVersion>
```

Comment:
Will this be auto-updated by darc?

Also, is there some way we can specify the version once and have it appear both here and in Version.Details.xml?

Reply:
> Will this be auto-updated by darc?

Yes, I expect this to be updated by our existing arcade->msbuild darc subscription.

> Also, is there some way we can specify the version once and have it appear both here and in Version.Details.xml?

I am not aware of one. Darc's design requires having it in two places updated by tooling.